show/fgnhg.py

```diff
@@ -52,61 +52,78 @@ def active_hops(nhg):
-                if ":" in nhg_prefix:
-                    for key in table_keys:
-                        mod_key = key.split("|")[1].split("/")[0]
-                        mod_nhg_prefix = nhg_prefix.split("/")[0]
-                        if ipaddress.ip_address(mod_key).exploded == ipaddress.ip_address(mod_nhg_prefix).exploded:
-                            t_dict = state_db.get_all(state_db.STATE_DB, key)
-                    nhg_prefix = "FG_ROUTE_TABLE|" + nhg_prefix
-                else:
-                    nhg_prefix = "FG_ROUTE_TABLE|" + nhg_prefix
-                    t_dict = state_db.get_all(state_db.STATE_DB, nhg_prefix)
-
+        nhip_prefix_map = {}
+        header = ["FG NHG Prefix", "Active Next Hops"]
+        try:
+            fg_nhg_member_table = config_db.get_table('FG_NHG_MEMBER')
+        except BaseException as e: 
+            ctx.fail("FG_NHG_MEMBER entries not present in config_db")
+            exit()
+        alias_list = []
+        nexthop_alias = {}
+        output_list = []
+
+        for nexthop, nexthop_metadata in fg_nhg_member_table.items():
+            alias_list.append(nexthop_metadata['FG_NHG'])
+            nexthop_alias[nexthop] = nexthop_metadata['FG_NHG']
+        if nhg not in alias_list:
+            ctx.fail ("Please provide a valid NHG alias")
+        else:
+            for nhg_prefix in table_keys:
+                t_dict = state_db.get_all(state_db.STATE_DB, nhg_prefix)
                 vals = sorted(set([val for val in t_dict.values()]))
 
                 for nh_ip in vals:
-                    if nhg_prefix in output_dict:
-                        output_dict[nhg_prefix].append(nh_ip.split("@")[0])
+                    nhip_prefix_map[nh_ip.split("@")[0]] = nhg_prefix
+                   
+                    if nh_ip.split("@")[0] in nexthop_alias:
+                        if nexthop_alias[nh_ip.split("@")[0]] == nhg:
+                            output_list.append(nh_ip.split("@")[0])
                     else:
-                        output_dict[nhg_prefix] = [nh_ip.split("@")[0]]
+                        ctx.fail ("state_db and config_db have FGNHG prefix config mismatch. Check device config!");
+                        sys.exit(1)                        
+                output_list = sorted(output_list)
+            if not output_list:
+                ctx.fail ("FG_ROUTE table likely does not contain the required entries")
+                sys.exit(1)
 
-                nhg_prefix_report = (nhg_prefix.split("|")[1])
-                formatted_nhps = ','.replace(',', '\n').join(output_dict[nhg_prefix])
-                table.append([nhg_prefix_report, formatted_nhps])
-                click.echo(tabulate(table, header, tablefmt="grid"))
+            nhg_prefix_report = nhip_prefix_map[output_list[0]].split("|")[1]
+            formatted_output_list = ','.replace(',', '\n').join(output_list)
+            table.append([nhg_prefix_report, formatted_output_list])
```

Comment:
@kktheballer can you please also align `tablefmt` for all FG CLI to make it SONiC common?

**Example:**
```
root@sonic:/home/admin# show interfaces status
      Interface            Lanes    Speed    MTU    FEC    Alias             Vlan    Oper    Admin    Type    Asym PFC
---------------  ---------------  -------  -----  -----  -------  ---------------  ------  -------  ------  ----------
      Ethernet0          0,1,2,3     100G   9100    N/A     etp1           routed      up       up     N/A         N/A
      Ethernet4          4,5,6,7     100G   9100    N/A     etp2           routed      up       up     N/A         N/A
      Ethernet8        8,9,10,11     100G   9100    N/A     etp3           routed      up       up     N/A         N/A
     Ethernet12      12,13,14,15     100G   9100    N/A     etp4           routed      up       up     N/A         N/A
     Ethernet16      16,17,18,19     100G   9100    N/A     etp5           routed      up       up     N/A         N/A
     Ethernet20      20,21,22,23     100G   9100    N/A     etp6           routed      up       up     N/A         N/A
     Ethernet24      24,25,26,27     100G   9100    N/A     etp7           routed      up       up     N/A         N/A
     Ethernet28      28,29,30,31     100G   9100    N/A     etp8           routed      up       up     N/A         N/A
     Ethernet32      32,33,34,35     100G   9100    N/A     etp9           routed      up       up     N/A         N/A
     Ethernet36      36,37,38,39     100G   9100    N/A    etp10           routed      up       up     N/A         N/A
     Ethernet40      40,41,42,43     100G   9100    N/A    etp11           routed      up       up     N/A         N/A
     Ethernet44      44,45,46,47     100G   9100    N/A    etp12           routed      up       up     N/A         N/A
     Ethernet48      48,49,50,51     100G   9100    N/A    etp13           routed      up       up     N/A         N/A
     Ethernet52      52,53,54,55     100G   9100    N/A    etp14           routed      up       up     N/A         N/A
     Ethernet56      56,57,58,59     100G   9100    N/A    etp15           routed      up       up     N/A         N/A
     Ethernet60      60,61,62,63     100G   9100    N/A    etp16           routed      up       up     N/A         N/A
     Ethernet64      64,65,66,67     100G   9100    N/A    etp17           routed      up       up     N/A         N/A
     Ethernet68      68,69,70,71     100G   9100    N/A    etp18           routed      up       up     N/A         N/A
     Ethernet72      72,73,74,75     100G   9100    N/A    etp19           routed      up       up     N/A         N/A
     Ethernet76      76,77,78,79     100G   9100    N/A    etp20           routed      up       up     N/A         N/A
     Ethernet80      80,81,82,83     100G   9100    N/A    etp21           routed      up       up     N/A         N/A
     Ethernet84      84,85,86,87     100G   9100    N/A    etp22           routed      up       up     N/A         N/A
     Ethernet88      88,89,90,91     100G   9100    N/A    etp23           routed      up       up     N/A         N/A
     Ethernet92      92,93,94,95     100G   9100    N/A    etp24           routed      up       up     N/A         N/A
     Ethernet96      96,97,98,99     100G   9100    N/A    etp25           routed      up       up     N/A         N/A
    Ethernet100  100,101,102,103     100G   9100    N/A    etp26           routed      up       up     N/A         N/A
    Ethernet104  104,105,106,107     100G   9100    N/A    etp27           routed      up       up     N/A         N/A
    Ethernet108  108,109,110,111     100G   9100    N/A    etp28           routed      up       up     N/A         N/A
    Ethernet112  112,113,114,115     100G   9100    N/A    etp29           routed      up       up     N/A         N/A
    Ethernet116  116,117,118,119     100G   9100    N/A    etp30           routed      up       up     N/A         N/A
    Ethernet120  120,121,122,123     100G   9100    N/A    etp31  PortChannel0001      up       up     N/A         N/A
    Ethernet124  124,125,126,127     100G   9100    N/A    etp32  PortChannel0002      up       up     N/A         N/A
PortChannel0001              N/A     100G   9100    N/A      N/A           routed    down     down     N/A         N/A
PortChannel0002              N/A     100G   9100    N/A      N/A           routed    down     down     N/A         N/A
```